src/app/component/admin/tag-group/list/list.component.ts

```diff
@@ -57,6 +56,8 @@ export class AdminTagGroupsComponent extends PagedTableTemplate<
         model: tagGroup,
       }))
     );
+
+    this.filterKey = "groupIdentifier";
```

Comment:
If this does what I think it does, probably better to be `tag.text`

Reply:
TagGroup does not have a text parameter